chart/templates/flower/flower-deployment.yaml

```
@@ -73,7 +73,11 @@ spec:
         - name: flower
           image: {{ template "flower_image" . }}
           imagePullPolicy: {{ .Values.images.flower.pullPolicy }}
-          args: ["bash", "-c", "airflow celery flower || airflow flower"]
+          {{- if semverCompare ">=2.0.0" .Values.airflowVersion }}
+          args: ["bash", "-c", "airflow celery flower"]
```

```suggestion
          args: ["airflow", "celery", "flower"]
```

Or if we really need bash (e.g. #13526 is still a problem):

```suggestion
          args: ["bash", "-c", "exec airflow celery flower"]
```

---

Will take care of that in a separate PR, trying to narrow the scope of this PR -- so keeping it as it was